website/layouts/index.html

```diff
@@ -9,6 +9,11 @@
     </header>
     
       <main role="main" class="td-main">
+        <section class="of-section-page-message">
+          <h1 class="of-heading of-heading--lg of-section-page-message__title"> Black lives matter.</h1>
+        <p>We stand in solidarity with the Black community.<br>Racism is unacceptable.<br> It conflicts with the <a href="https://git.k8s.io/community/values.md">core values of the Kubernetes project</a> and our community does not tolerate it.</p>
```

Comment:
```suggestion
        <p>We stand in solidarity with the Black community.<br>Racism is unacceptable.<br> It conflicts with the <a href="https://git.k8s.io/community/values.md">core values of the Operator SDK project</a> and our community does not tolerate it.</p>
```

Reply:
We are using the Kubernetes statement verbatim. I think it makes sense because we are a part of the larger community.

Reply:
Didn't realize this until @camilamacedo86 pointed this out, but the "Kubernetes project" wording is a bit specific, and IMO distinct from this project.

I would suggest the following tweak:
```suggestion
        <p>We stand in solidarity with the Black community.<br>Racism is unacceptable.<br> It conflicts with the <a href="https://git.k8s.io/community/values.md">core values of the Kubernetes community</a> and we do not tolerate it.</p>
```